src/apps/explorer/components/common/ExplorerTabs/ExplorerTab.tsx

```diff
@@ -38,11 +38,16 @@ const StyledTabs = styled.div`
 
   .tab-extra-content {
     width: 100%;
-
     @media ${MEDIA.mobile} {
       display: none; /* for now we can hide the extra-content on mobiles */
     }
   }
+  @media ${MEDIA.mobile} {
+    > div > div.tab-content {
```

Comment:
Keep the code dry and move the media query inside line 35. This way you don't have to repeat the selector and the selectors are more consolidated.

Reply:
Also, is `.tab-content` and `.tablist` exclusively used on the page at the specific DOM tree positions? If so, we could simplify the selector:

from:
`> div > div.tab-content`

to:
`.tab-content`

My approach with using specifity when using vanilla HTML elements (div, span, etc.) is to do it only when I know the vanilla elements will not be re-used else-where *or* are scoped already inside a styled component:

For example `${tab} > b`
`<Tab><b>Example</b></Tab>`

In the current code I could see us having dryer code by doing:
- Turn `.tablist` into a styled component
- Then you only need one nested rule for the button (`> button`)
- Also, the styled component now can be re-used else where in the app by making it an `export`.